updating/updating-restricted-network-cluster/mirroring-image-repository.adoc

```diff
@@ -13,19 +13,39 @@ You must mirror container images onto a mirror registry before you can update a
 Your mirror registry must be running at all times while the cluster is running.
 ====
 
-There are two methods for mirroring images onto a mirror registry:
+The following steps outline the high-level workflow on how to mirror images to a mirror registry:
 
-* xref:../../updating/updating-restricted-network-cluster/mirroring-image-repository.adoc#mirroring-ocp-resources-ocmirror[Using the oc-mirror OpenShift CLI (`oc`) plugin]
+. Install the OpenShift CLI (`oc`) on all devices being used to retrieve and push release images.
 
-* xref:../../updating/updating-restricted-network-cluster/mirroring-image-repository.adoc#update-mirror-repository-adm-release-mirror_mirroring-ocp-image-repository[Using the `oc adm release mirror` command]
+. Download the registry pull secret and add it to your cluster.
 
-Compared to using the `oc adm release mirror`command, the oc-mirror plugin has the following advantages:
+. If you use the xref:../../updating/updating-restricted-network-cluster/mirroring-image-repository.adoc#mirroring-ocp-resources-ocmirror[oc-mirror OpenShift CLI (`oc`) plugin]:
+
+.. Install the oc-mirror OpenShift CLI plugin on all devices being used to retrieve and push release images.
+
+.. Create an image set configuration file for the plugin to use when determining which release images to mirror. You can edit this configuration file later to change which release images the plugin mirrors.
+
+.. Mirror your targeted release images directly to a mirror registry, or to removable media and then to a mirror registry.
+
+.. Install the `ImageContentSourcePolicy` and `CatalogSource` resources that oc-mirror generated into the cluster.
```

Comment:
FYI: The [ICSP is deprecated](https://github.com/openshift/openshift-docs/pull/56250/files#diff-b65563ec32c0710c0c09470e49a13b38d89f75bf7e6c09baa17443723f5e329bR13) as of 4.13 and is planned to be removed at some point.